etc/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -91,7 +91,7 @@
 # also can make the collection of support bundles more useful.
 #
 - lineinfile:
-    dest: /etc/systemd/journald.conf
+    dest: /etc/systemd/journald.conf.d/override.conf
```

Comment:
Is it not preferable to ship a packaged override.conf file for this rather than ansible so that we can make changes to that file without adding ansible logic to undo existing changes?

Reply:
definitely, thanks! idk why I didn't think of that to begin with.